examples/with-reflexjs/.babelrc

```diff
@@ -0,0 +1,13 @@
+{
+  "presets": [
+    [
+      "next/babel",
+      {
+        "preset-react": {
+          "runtime": "automatic",
+          "importSource": "reflexjs"
```

Comment:
Why is this needed?

Reply:
This sets a custom pragma to convert style props to sx props. So that you don't need to import it for each individual files.

https://reflexjs.org/docs/installation/nextjs#add-the-babel-preset